tests/Plugins/GitHubCommunityStandards_EndToEndTest.py

```diff
@@ -61,7 +61,23 @@ def test_CodeOfConduct(self, pat_args, snapshot):
         """Test if a CODE_OF_CONDUCT file exists in the repository."""
         result = CliRunner().invoke(
             app,
-            pat_args + ["--GitHubCommunityStandards-CodeOfConduct-exists"],
+            pat_args
+            + [
+                "--GitHubCommunityStandards-CodeOwners-exists",
```

Comment:
These command line arguments don't look right to me. For flags (boolean values), the value is the default if not provided and the negated default if the value is provided. In other words, specifying `false` is not required.

Also, the naming seems to be backwards. Since the flag is provided, it means that either:

A) We don't care if the file exists or not

or

B) We should enforce that the file doesn't exist

I'm not sure if we should implement A or B. Either way, the naming on the argument will need to change.